src/components/live-traffic.vue

```diff
@@ -105,9 +105,7 @@ export default class extends DashboardWidget {
     this.BoxedCountFlight = Object.keys(flights).length;
     this.BoxedCountAirline = airlines.size;
     if (this.BoxedCountAirline !== 0) {
-      const maxSpeedFlight = flights.reduce((max: Flight, f: Flight) => {
-        return ( f.speed > max.speed ? f : max );
-      });
+      const maxSpeedFlight = flights.reduce((max: Flight, f: Flight) => f.speed > max.speed ? f : max);
```

Comment:
I mean, if flights is an empty list, this throws an exception. Maybe we can define a default in this case and pass it as the second argument of reduce.
The default may be `{ speed: 0, icaoNumber: undefined }`.

Reply:
Or, we can check the flights.size > 0 above.